etc/sca/windows/cis_aws_linux_rcl.yml

```diff
@@ -0,0 +1,1005 @@
+# Security Configuration assessment
+# CIS Checks for RHEL 7
+# Copyright (C) 2015-2019, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation
+#
+# Based on:
+# Center for Internet Security Red Hat Enterprise Linux 7 Benchmark v2.2.0 - 12-27-2017
+
+policy:
+  id: "cis_rhel7_aws"
+  file: "cis_aws_linux_rcl.yml"
+  name: "CIS Benchmark for Red Hat Enterprise Linux 7"
+  description: "This document provides prescriptive guidance for establishing a secure configuration posture for Red Hat Enterprise Linux 7 systems running on x86 and x64 platforms. This document was tested against Red Hat Enterprise Linux 7.4."
+  references:
+    - https://www.cisecurity.org/cis-benchmarks/
+
+requirements:
+  title: "Check RHEL7 version"
+  description: "Requirements for running the SCA scan against RHEL 7."
+  condition: "any required"
+  rules:
+     - 'f:/etc/redhat-release -> r:^Red Hat Enterprise Linux \S+ release 7;'
+     - 'f:/etc/redhat-release -> IN r:^CentOS && r:release 7;'
+     - 'f:/etc/redhat-release -> IN r:^Cloud && r:release 7;'
+     - 'f:/etc/redhat-release -> IN r:^Oracle && r:release 7;'
+     - 'f:/etc/redhat-release -> IN r:^Better && r:release 7;'
+     - 'f:/etc/redhat-release -> IN r:^OpenVZ && r:release 7;'
+     - 'f:/etc/system-release -> IN r:^Amazon && r:release 2;'
+
+variables:
+  $sshd_file: /etc/ssh/sshd_config;
+# RC scripts location
+  $rc_dirs: /etc/rc.d/rc2.d,/etc/rc.d/rc3.d,/etc/rc.d/rc4.d,/etc/rc.d/rc5.d;
+
+checks:
+# 1.1.2 /tmp: partition
+ - id: 6500
```

Comment:
IDs are already in use https://github.com/wazuh/wazuh/blob/67ecaa64da8c6b92b2aad16f48b0ba0cc11ab007/etc/sca/sles/12/cis_sles12_linux.yml#L33.

Please review IDs of all the included policies.